src/Downloads.jl

```diff
@@ -423,4 +423,21 @@ function content_length(headers::Union{AbstractVector, AbstractDict})
     return nothing
 end
 
+"""
+    default_downloader!(
+        downloader = <none>
+    ) 
+
+        downloader :: Downloader
+
+Set the default `Downloader` that is used when none is provided.  
```

Comment:
Maybe:
```suggestion
Set the default `Downloader`. If no argument is provided, resets the default downloader so that a fresh one is created the next time the default downloader is needed.
```
The feature to just reset it can be useful; e.g. if you encounter a transient error from AWS, they [suggest](https://docs.aws.amazon.com/sdkref/latest/guide/feature-retry-behavior.html) you do not re-use the connection, which I believe with Downloads.jl is achieved by getting a fresh Downloader object.

Reply:
Agreed/Thanks -- upon re-reading, not entirely sure what my description was trying to convey...